### PR TITLE
Inclass fix + auth style

### PR DESCRIPTION
**Pull request by cryptokoala4**

### CODE REVIEW

File: public/js/torrents/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$(function(){
```
Comment (f3r) [inline public/js/torrents/index.js:1]: This is halfway there:

On how to integrate this like Dennis suggested, pls check: https://github.com/wdi-hk-9/WDI-Express/compare/frontend-api

image

Comment (cryptokoala4) [inline public/js/torrents/index.js:1]: @fer amended

**Comment by cryptokoala4** [comment]
amended~

**Comment by cryptokoala4** [comment]
@f3r Updated navbar to be responsive. Working on body to hold content next.

**Pull request merged**